terraform/addons/byo-s3-logging-destination-firehose/target-account/main.tf

```diff
@@ -0,0 +1,195 @@
+variable "osquery_results_bucket" {
+  type        = string
+  description = "name of the bucket to store osquery results logs"
+}
+
+variable "osquery_status_bucket" {
+  type        = string
+  description = "name of the bucket to store osquery status logs"
+}
+
+variable "fleet_iam_role_arn" {
+  type        = string
+  description = "the arn of the fleet role that firehose will assume to write data to your bucket"
+  default     = "arn:aws:iam::123456789123:role/fix-me"
```

Comment:
If its going to be a "fixme" default, why not just make the argument required?

Reply:
made required argument